lib/transport_web/api/serializers/dataset_serializer.ex

```diff
@@ -4,13 +4,8 @@ defmodule TransportWeb.API.DatasetSerializer do
   """
 
   use TransportWeb, :serializer
-  alias BSON.ObjectId
-  alias Transport.ReusableData.Dataset
 
-  location "/datasets/:slug/"
-  attributes [:title, :slug, :coordinates, :validations]
+  location "/datasets/:id/"
```

Comment:
Same, this is our front's representation, it not the same context than our internal relationship with data.gouv.fr.